<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScmEditorOptions</class>
 <widget class="QWidget" name="ScmEditorOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="useInactiveEffects">
     <property name="text">
      <string>Apply &amp;effects to inactive windows</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="inactiveSelectionEffect">
     <property name="text">
      <string>Use different colors for in&amp;active selections</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="shadeSortedColumn">
     <property name="text">
      <string>Shade sorted column &amp;in lists</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="toolTip">
      <string>Shading of frames and lighting (&quot;3D&quot;) effects</string>
     </property>
     <property name="title">
      <string>Shading</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="horizontalSpacing">
       <number>12</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="0" column="1">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Minimum</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>198</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Maximum</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Contrast</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QSlider" name="contrastSlider">
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="pageStep">
         <number>5</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
